core/server/data/migration/fixtures/005/05-transform-dates-into-UTC.js

```diff
@@ -0,0 +1,132 @@
+                    post.set('updated_at', moment(post.get('updated_at')).add(ServerTimezoneOffset, 'minutes').toDate());
+
+                    options.id = post.get('id');
+                    return models.Post.edit(post.toJSON(), options);
+                }).then(function () {
+                    logger.info(messagePrefix + 'Updated datetime fields for Posts!');
+                });
+            });
+        },
+        function () {
+            return models.Users.forge().fetch().then(function (result) {
+                if (result.models.length === 0) {
+                    logger.warn(messagePrefix + 'No Users found');
+                    return;
+                }
+
+                return Promise.map(result.models, function mapper(user) {
+                    if (user.get('last_login')) {
+                        user.set('last_login', moment(user.get('last_login')).add(ServerTimezoneOffset, 'minutes').toDate());
+                    }
+
+                    if (user.get('updated_at')) {
+                        user.set('updated_at', moment(user.get('updated_at')).add(ServerTimezoneOffset, 'minutes').toDate());
+                    }
+
+                    user.set('created_at', moment(user.get('created_at')).add(ServerTimezoneOffset, 'minutes').toDate());
+
+                    options.id = user.get('id');
+                    return models.User.edit(user.toJSON(), options);
+                }).then(function () {
+                    logger.info(messagePrefix + 'Updated datetime fields for Users!');
+                });
+            });
+        },
+        function () {
+            return models.Subscribers.forge().fetch().then(function (result) {
+                if (result.models.length === 0) {
+                    logger.warn(messagePrefix + 'No Subscribers found');
+                    return;
+                }
+
+                return Promise.map(result.models, function mapper(subscriber) {
+                    if (subscriber.get('unsubscribed_at')) {
+                        subscriber.set('unsubscribed_at', moment(subscriber.get('unsubscribed_at')).add(ServerTimezoneOffset, 'minutes').toDate());
+                    }
+
+                    if (subscriber.get('updated_at')) {
+                        subscriber.set('updated_at', moment(subscriber.get('updated_at')).add(ServerTimezoneOffset, 'minutes').toDate());
+                    }
+
+                    subscriber.set('created_at', moment(subscriber.get('created_at')).add(ServerTimezoneOffset, 'minutes').toDate());
+
+                    options.id = subscriber.get('id');
+                    return models.Subscriber.edit(subscriber.toJSON(), options);
+                }).then(function () {
+                    logger.info(messagePrefix + 'Updated datetime fields for Subscribers!');
+                });
+            });
+        },
+        function () {
+            return models.Settings.forge().fetchAll().then(function (result) {
```

Comment:
> nextUpdateCheck: agree, the update check is not important, its just a check with a notification, so it can happen earlier or later, we don't care.

👍 

> tokens: if we don't change the logic at all for tokens, it will work, because 1. tokens are stored as bigInteger and are not transformed at all. 2. they are created and compared with Date.now, which is local. the big disadvantage is inconsistency. i would prefer changing the logic for it, so that they are treated as UTC.

This sounds sensible, but I'm happy to do it either now, or as a separate fix after the main part is done?